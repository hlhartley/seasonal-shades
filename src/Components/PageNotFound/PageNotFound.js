import React from 'react';

const PageNotFound = () => {
    return(
        <div className='pagenotfound-container'>
            <i className="fas fa-exclamation-triangle"></i> 
            <p className='pagenotfound-text'><b>404:</b> Page not found</p>
        </div>
    )
}

export default PageNotFound;