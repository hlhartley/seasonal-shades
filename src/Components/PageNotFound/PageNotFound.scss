.pagenotfound-container {
    align-items: center;
    display: flex;
    flex-direction: column;
    justify-content: center;
    margin-top: 35px;

    .pagenotfound-text {
        font-size: 20px;
    }

    b {
        font-size: 20px;
    }
}

.fa-exclamation-triangle {
    color: $grapefruit-color;
    font-size: 40px;
}