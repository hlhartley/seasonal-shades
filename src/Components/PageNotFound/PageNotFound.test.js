import React from 'react';
import PageNotFound from './PageNotFound';
import { shallow } from 'enzyme';

describe('PageNotFound Component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<PageNotFound />)
    });

    it('should have a snapshot with the data passed in corectly', () => {
        expect(wrapper).toMatchSnapshot()
    });
});