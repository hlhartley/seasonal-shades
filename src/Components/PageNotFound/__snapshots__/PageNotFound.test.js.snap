// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Component should have a snapshot with the data passed in corectly 1`] = `
<div
  className="pagenotfound-container"
>
  <i
    className="fas fa-exclamation-triangle"
  />
  <p
    className="pagenotfound-text"
  >
    <b>
      404:
    </b>
     Page not found
  </p>
</div>
`;
